app/views/directives/bind-service/delete-binding-result.html

```diff
@@ -0,0 +1,31 @@
+<div>
+  <div ng-if="!ctrl.error">
+    <h3 class="mar-top-none">
+      Binding for the following has been deleted:
+    </h3>
+
+    <div
+      ng-if="ctrl.appsForBinding(ctrl.selectedBinding)"
+      ng-repeat="appForBinding in ctrl.appsForBinding(ctrl.selectedBinding)">
+      {{appForBinding.metadata.name}} <small class="text-muted">&ndash; {{ appForBinding.kind | humanizeKind : true}}</small>
+    </div>
+    <div ng-if="!(ctrl.appsForBinding(ctrl.selectedBinding))">
+      {{ctrl.selectedBinding}} <small class="text-muted">&ndash; Secret</small>
```

Comment:
I'll swap this & leave a `TODO:` because its weird to select "secret" then see "binding" deleted, agree. :/